packages/testcontainers/src/container-runtime/clients/container/podman-container-client.ts

```diff
@@ -22,32 +23,52 @@ export class PodmanContainerClient extends DockerContainerClient {
       execOptions.User = opts.user;
     }
 
-    const chunks: string[] = [];
+    const outputChunks: string[] = [];
+    const stdoutChunks: string[] = [];
+    const stderrChunks: string[] = [];
+
     try {
       if (opts?.log) {
         log.debug(`Execing container with command "${command.join(" ")}"...`, { containerId: container.id });
       }
 
       const exec = await container.exec(execOptions);
       const stream = await this.demuxStream(container.id, await exec.start({ stdin: true, Detach: false, Tty: true }));
```

Comment:
Perhaps it is no longer necessary to demux the stream here, if you doing it on line 41. If this line can be removed, then I don't think there is any difference between the Docker and Podman implementations? If so we can remove the whole `podman-container-client` class 🚀

Reply:
I agree as well. I just disabled the exec method in the Podman client implementation and replaced it with the Docker client implementation's exec method. I confirmed that the test cases pass without any issues.

Should I proceed with modifying the code to remove the Podman client implementation and commit the changes?

Reply:
> Should I proceed with modifying the code to remove the Podman client implementation and commit the changes?

That would be great